### ISSUE TITLE
[a0] friend_overlap() how to define user1 user2

### ISSUE THREAD

**Issue by ppttzhu** [iit-cs579/main#481]
Notice that in test case output:
`>>> friend_overlap([`
    `...     {'screen_name': 'a', 'friends': ['1', '2', '3']},`
    `...     {'screen_name': 'b', 'friends': ['2', '3', '4']},`
    `...     {'screen_name': 'c', 'friends': ['1', '2', '3']},`
    `...     ])`
    `[('a', 'c', 3), ('a', 'b', 2), ('b', 'c', 2)]`

There is no ('c', 'a', 3), so clearly 'a' is user1 and 'c' is user2.
Are user1/user2 decided by its index in input list or in alphabetic order?

In other words, not sure what's the output in this test case:
`>>> friend_overlap([`
    `...     {'screen_name': 'c', 'friends': ['1', '2', '3']},`
    `...     {'screen_name': 'b', 'friends': ['2', '3', '4']},`
    `...     {'screen_name': 'a', 'friends': ['1', '2', '3']},`
    `...     ])`

**(This issue is not important because we will sort input list in alphabetical order. )**

**Comment by aronwc** [comment]
The output for your example would be:
`[('c', 'a', 3), ('b', 'a', 2), ('c', 'b', 2)]`

In this case, it is `('c', 'a', 3)` instead of `('a', 'c', 3)` because user 'c' appeared first in the input list.
